bin/asciibinder

```diff
@@ -17,7 +18,15 @@ def call_generate(distro,page=nil)
   end
 end
 
-SUB_COMMANDS = %w{help build watch package clean create}
+def repo_check(repo_dir)
+  ['.git','_build_cfg.yml','_distro_map.yml','_templates'].each do |file|
+    if not File.exist?(File.join(repo_dir, file))
```

Comment:
We gotta get some rubocop in here :D  Things like "not", "and", "or" and generally discouraged, and aren't really used in general practice.  It;s find for this PR, but let's discuss